docs/Manual/Deployment/Kubernetes/Drain.md

```diff
@@ -0,0 +1,444 @@
+the internal name of the _DBserver_ is `PRMR-wbsq47rz`. Note that `PRMR`
+must be all capitals since pod names are always all lower case. So, we
+could use the body:
+
+```JSON
+{"server":"PRMR-wbsq47rz"}
+```
+
+The API call will return immediately with a body like this:
+
+```JSON
+{"error":false,"id":"38029195","code":202}
+```
+
+The given `id` in this response can be used to query the outcome or
+completion status of the clean out server job with this API:
+
+```
+GET /_admin/cluster/queryAgencyJob?id=38029195
+```
+
+… which will return a body like this:
+
+```JSON
+{
+  "error": false,
+  "id": "38029195",
+  "status": "Pending",
+  "job": {
+    "timeCreated": "2019-02-21T10:42:14.727Z",
+    "server": "PRMR-wbsq47rz",
+    "timeStarted": "2019-02-21T10:42:15Z",
+    "type": "cleanOutServer",
+    "creator": "CRDN-rxtu5pku",
+    "jobId": "38029195"
+  },
+  "code": 200
+}
+```
+
+It indicates that the job is still ongoing (`"Pending"`). As soon as
+the job has completed, the answer will be:
+
+```JSON
+{
+  "error": false,
+  "id": "38029195",
+  "status": "Finished",
+  "job": {
+    "timeCreated": "2019-02-21T10:42:14.727Z",
+    "server": "PRMR-e6hbjob1",
+    "jobId": "38029195",
+    "timeStarted": "2019-02-21T10:42:15Z",
+    "timeFinished": "2019-02-21T10:45:39Z",
+    "type": "cleanOutServer",
+    "creator": "CRDN-rxtu5pku"
+  },
+  "code": 200
+}
+```
+
+From this moment on the _DBserver_ can no longer be used to move
+shards to. At the same time, it will no longer hold any data of the
+cluster.
+
+Now the drain operation involving a node with this pod on it is
+completely risk-free, even with a small grace period.
+
+## Performing the drain
+
+After all above [checks before a node drain](#things-to-check-in-arangodb-before-a-node-drain)
+have been done successfully, it is safe to perform the drain
+operation, similar to this command:
+
+```bash
+kubectl drain gke-draintest-default-pool-394fe601-glts --delete-local-data --ignore-daemonsets --grace-period=300
+```
+
+As described above, the options `--delete-local-data` for ArangoDB and
+`--ignore-daemonsets` for other services have been added. A `--grace-period` of
+300 seconds has been chosen because for this example we are confident that all the data on our _DBServer_ pod
+can be moved to a different server within 5 minutes. Note that this is
+**not saying** that 300 seconds will always be enough, regardless of how
+much data is stored in the pod, your mileage may vary, moving a terabyte
+of data can take considerably longer!
+
+If the optional step of
+[cleaning out a DBserver manually](#clean-out-a-dbserver-manually-optional)
+has been performed beforehand, the grace period can easily be reduced to 60
+seconds - at least from the perspective of ArangoDB, since the server is already
+cleaned out, so it can be dropped readily and there is still no risk.
+
+At the same time, this guarantees now that the drain is completed
+approximately within a minute.
```

Comment:
Sounds like cleaning out the server manually is what we should recommend, yet we say it's optional. What are the other options? The problem I see with any grace period is that you never know if it's too long or too short, and that the draining process never finish before the end of the grace period...